company_linkage/airtable_schemas/company_annotation/parentage.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "description": "CSET identifier for the organization",
+    "mode": "REPEATED",
+    "name": "CSET_id",
+    "type": "INTEGER"
+  },
+  {
+    "description": "CSET identifier for the parent organization",
+    "mode": "REPEATED",
+    "name": "parent_id",
+    "type": "INTEGER"
+  },
+  {
+    "description": "Name of the parent organization",
+    "mode": "NULLABLE",
+    "name": "parent_name",
+    "type": "STRING"
+  },
+  {
+    "description": "Parent through acquisition",
+    "mode": "NULLABLE",
+    "name": "is_acquisition_parent",
```

Comment:
If you're renaming variables like this it's going to break aggregate_organizations.py

The choice here is to either not rename (leave it as parent_acquisition) or modify aggregate_organizations.py, which I don't see changed here.

I don't know if any other variable names got changed like this; this is the one that stood out to me, but any that did would need this dealt with.